### PR TITLE
✨  Add system support for Progress

### PR DESCRIPTION
**Pull request by mmartinsoliveira**
# ✨ Add system support for Progress

### Good News

News props for Progress Component:

- `backgroundColor` 
- `spacing`
- `border`
- `elevation`
- `layout`

### Screenshot

image

### CODE REVIEW

**Review by ericcleao — APPROVED** [review]

File: packages/yoga/src/Progress/native/Progress.jsx [hunk 1]
```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import { string, number, shape, oneOfType, oneOf } from 'prop-types';
+import {
+  backgroundColor,
+  border,
+  elevation,
+  positions,
+  display,
+  spacing as Spacing,
```
Comment (ggdaltoso) [inline packages/yoga/src/Progress/native/Progress.jsx]:
```suggestion
  spacing as systemSpacing,
```

just to not confuse JSX parser

**Review by ggdaltoso — APPROVED** [review]
Great job!!!

File: packages/yoga/src/Progress/native/Progress.jsx [hunk 2]
```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import styled from 'styled-components';
 import { string, number, shape, oneOfType, oneOf } from 'prop-types';
+import {
+  backgroundColor,
+  border,
```
Comment (ericcleao) [inline packages/yoga/src/Progress/native/Progress.jsx:6]: @mmartinsoliveira here we are importing only the `border` prop, but on the tests, we are testing the `borderRadius`.
Should it be `borders` to get all system props associated to the border or just be the `border` one?

Comment (ggdaltoso) [reply packages/yoga/src/Progress/native/Progress.jsx]: Solved, @ericcleao

**Pull request merged**